src/components/Asset/AssetActions/Pool/Remove/index.module.css

```diff
@@ -50,25 +50,30 @@
 }
 
 .output {
-  composes: output from '../Add/Output.module.css';
+  border-top: 1px solid var(--border-color);
+  padding: var(--spacer);
+  padding-bottom: calc(var(--spacer) / 1.5);
+  display: grid;
+  gap: var(--spacer);
+  grid-template-columns: 1fr 1fr;
+  margin-left: -2rem;
+  margin-right: -2rem;
 }
 
-.output [class*='token'] {
-  white-space: nowrap;
+.output p {
+  font-weight: var(--font-weight-bold);
+  margin-bottom: calc(var(--spacer) / 8);
+  font-size: var(--font-size-small);
 }
 
-.output [class*='token'] > figure {
-  display: inline-block;
+.output [class*='token'] {
+  white-space: normal;
```

Comment:
Was introducing wrapping intended here? This would make token image + value + symbol wrap to next line, whereas original intention was for them to be always on one line. Wrapping after token image would be fine in some instances I guess but see we are hiding it anyway in this case. Effectively making value and symbol wrap which should be prevented at all times